packages/lib/src/vis/rgb/RgbMesh.tsx

```diff
@@ -0,0 +1,48 @@
+import type { NdArray } from 'ndarray';
+import { useMemo } from 'react';
+
+import VisMesh from '../shared/VisMesh';
+import { getUniforms, VERTEX_SHADER } from '../utils';
+import { getDataTexture3D } from './utils';
+
+interface Props {
+  values: NdArray<Uint8Array | Uint8ClampedArray | Float32Array>;
+  bgr?: boolean;
+}
+
+function RgbMesh(props: Props) {
+  const { values, bgr = false } = props;
+
+  const dataTexture = useMemo(() => getDataTexture3D(values), [values]);
+
+  const shader = {
+    uniforms: getUniforms({ data: dataTexture, bgr }),
+    vertexShader: VERTEX_SHADER,
+    fragmentShader: `
+      uniform highp sampler3D data;
+      uniform bool bgr;
+
+      varying vec2 coords;
+
+      void main() {
+        float red = texture(data, vec3(0., coords)).r;
+        float green = texture(data, vec3(0.5, coords)).r;
+        float blue = texture(data, vec3(1., coords)).r;
+
+        if (bgr) {
+          gl_FragColor = vec4(blue, green, red, 1.);
+        } else {
+          gl_FragColor = vec4(red, green, blue, 1.);
+        }
+      }
+    `,
+  };
+
+  return (
+    <VisMesh scale={[1, -1, 1]}>
```

Comment:
It would be funny to try [flipY](https://threejs.org/docs/index.html?q=DataT#api/en/textures/Texture.flipY) from the texture

Reply:
Hmm, it seems to be inverting the 3rd dimension as well (i.e. the channels), it's weird:

![image](https://user-images.githubusercontent.com/2936402/154233019-c12d2b4b-f28f-43b5-8dcf-b8ad1d2fc399.png)

Reply:
That being said, I managed to flip Y only in the shader. Pusing now.

Reply:
In this end, I don't think it is worth doing in the shader: it is more obscure than using the `scale` as in `HeatmapMesh`.

Reply:
Ah damn, sorry, I merged it 😞